app/components/UI/Navbar/index.js

```diff
@@ -1811,3 +1811,36 @@ export const getSettingsNavigationOptions = (title, themeColors) => {
     ...innerStyles,
   };
 };
+
+export function getStakeInputNavbar(navigation, themeColors) {
+  const innerStyles = StyleSheet.create({
+    headerButtonText: {
+      color: themeColors.primary.default,
+      fontSize: 14,
+      ...fontStyles.normal,
+    },
+    headerStyle: {
+      backgroundColor: themeColors.background.default,
+      shadowColor: importedColors.transparent,
+      elevation: 0,
+    },
+  });
+  const title = 'Stake ETH';
```

Comment:
Should we swap this hardcoded English string with a local var to support other languages?

```suggestion
  const title = strings('stake.stake_eth') // or some other var name;
```

Reply:
Good idea!